src/pages/speaker-list/speaker-list.scss

```diff
@@ -1,5 +1,5 @@
 
-page-speaker-list {
+.page-speaker-list {
```

Comment:
This is the incorrect selector, see https://github.com/driftyco/ionic-conference-app/blob/master/src/pages/speaker-list/speaker-list.ts#L12